app/App/Features/ToDo/Containers/ToDoScreen.js

```diff
@@ -1,33 +1,42 @@
 // @flow
 import React, { useCallback, useState, useEffect } from 'react'
-import { View, Text, ImageBackground, Image, TouchableOpacity, FlatList } from 'react-native'
+import { View, Text, ImageBackground, Image, TouchableOpacity, FlatList, ActivityIndicator, Modal, Picker } from 'react-native'
 import { useDispatch, useSelector } from 'react-redux'
 
 import ToDo from '../Components/ToDo'
 import TogglableText from '../Components/TogglableText'
 
 import { actions as ToDosUIActions } from '../Redux/Ui'
 import ToDoEntitySelectors from '../Selectors/Entity'
-import ToDoUISelections from '../Selectors/Ui'
+import ToDoUISelections, { fetching } from '../Selectors/Ui'
 
 import styles from './ToDoScreen.style'
 import { Images } from '../../../Themes'
 
 import type { StackNavigationProp } from '@react-navigation/stack'
 
-import moment from 'moment'
+import moment from 'moment' //data-hora
+import { TextInput } from 'react-native-gesture-handler'
+import colors from '../../../Themes/Colors'
+import { values } from 'lodash'
+
+
 
 type Props = {
   navigation: StackNavigationProp
 }
 
+
 const ToDoScreen = ({ navigation }: Props) => {
   // Redux Actions
   const dispatch = useDispatch()
   const getToDos = useCallback(() => dispatch(ToDosUIActions.request()))
 
   // State
-  const [selectedFilterIndex, setFilterIndex] = useState(0)
+ const [selectedFilterIndex, setFilterIndex] = useState(0)
+ const [add, setAdd] = useState(false)
```

Comment:
Com propriedades booleanas, é recomendado seguir o padrão de adicionar um `is` ou `are` no início da frase. Por exemplo:
```
const [isAdding, setIsAdding] = useState(false)
const [isOpen, setIsOpen] = useState(false) 
```
ou se você preferir (com tipagem do Typescipt):
```
const [isAdding, setAddingStatus] = useState<boolean>(false)
const [isOpen, setOpenStatus] = useState<boolean>(false) 
```

Reply:
E seria interessante também deixar mais claro o que está sendo adicionado, aberto ou o que a data representa, por exemplo: 

```
const [isAddingSomething, setAddingSomething] = useState<boolean>(false)
const [isSomethingOpen, setIsSomethingOpen] = useState<boolean>(false)
const [reminderDate, setReminderDate] = useState<Date>(new Date())
```